File: python/Programmers_level/level1/문자열 내 p와 y의 개수.py
# 내 풀이
from collections import Counter
def solution(s):
    dict = Counter(s.lower())
    p_num = dict["p"]
    y_num = dict["y"] # dict에 없으면 0을출력해준다. Counter는 친절하다..
    return p_num == y_num


# 방법 2 -> count() 사용
def numPY(s):
    # 함수를 완성하세요
    return s.lower().count('p') == s.lower().count('y')

File: Javascript/Programmers_level/level1/문자열 내 p와 y의 개수.js
function solution(s) {
  let p = 0
  let y = 0
  for (item of s.toLowerCase()) {
    if (item === 'p') p += 1
    else if (item === 'y') y += 1
  }

  if (p === y) return true
  else if (p === 0 && y === 0) return true
  else return false
}
